### PR TITLE
[release-4.11] OCPBUGS-5882: Set upgradeability check throttling period to 2m

### PR DESCRIPTION
**Pull request by petr-muller**
Previously, the throttling reused the `minimumUpdateCheckInterval` value
which is derived from the full CVO minimum sync period. This value is
set between 2m and 4m at CVO startup. This period is unecessarily long
and bad for UX, things happen with a delay and our own testcase expects
upgradeability to be propagated in 3 minutes at worst.

Hardcode the throttling to 2m (lower bound of previous behavior) to
prevent flapping on flurries but allow changes to propagate
deterministically faster. We will still get a bit of non-determinisim
from sync periods and requeueing, so this change should not cause any
periodic API-hammering.

This is a partial backport of https://github.com/openshift/cluster-version-operator/pull/882. The `release-4.11` branch does not contain the fast-mode-with-failing-precondition code from https://github.com/openshift/cluster-version-operator/pull/808 so the refactoring commit from https://github.com/openshift/cluster-version-operator/pull/882 is not applicable nor needed.

### CODE REVIEW

**Comment by evakhoni** [comment]
OCPBUGS-5879 is verified    
/jira refresh

**Comment by evakhoni** [comment]
@petr-muller any changes expected to this PR? I can pre-merge verify if definitely not.

**Comment by petr-muller** [comment]
@evakhoni nope, no changes expected

/payload-aggregate periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-azure-upgrade 10

**Comment by evakhoni** [comment]
pre-merge verified in [comment-21590007](https://issues.redhat.com/browse/OCPBUGS-5882?focusedCommentId=21590007&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-21590007)  
/label qe-approved

**Comment by evakhoni** [comment]
@bandrade any idea who is available to set backport-risk-assessed on this one? tnx!

**Comment by petr-muller** [comment]
/hold

I'd like to have a better look into some of the results in https://github.com/openshift/cluster-version-operator/pull/885#issuecomment-1399637551

**Comment by petr-muller** [comment]
@evakhoni `backport-risk-assessed` is dev team lead's, so @LalatenduMohanty

**Comment by LalatenduMohanty** [comment]
Lets give it little more time i.e. at least a week after 4.12 backport is merged and then we can merge this.

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Comment by evakhoni** [comment]
> @evakhoni unfortunately I can't help here, the qe-approvals members are https://github.com/openshift/release/blob/721527e9122aea33aa225514d5f59a3ef3523291/core-services/prow/02_config/openshift/cluster-version-operator/_pluginconfig.yaml#L7-L9

well, Jia would not be back until summer, so we have @jianlinliu as the only allowed_user for now. 
anyway, tnx for trying!

**Comment by petr-muller** [comment]
@evakhoni perhaps we should add you to that list? I'm happy to merge such PR.

**Comment by evakhoni** [comment]
> @evakhoni perhaps we should add you to that list? I'm happy to merge such PR.

I think its something we need to discuss with my team first. (although this PR looks perfectly fine to me, speaking generally of being a cherry pick approver)

**Pull request merged**
